### DISCUSSION TITLE
404Have a problem after i publish my project on GitHub Pages.

### DISCUSSION THREAD

**Discussion by Vatafus** [community/community#101495] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have the current repo: https://github.com/Vatafus/AboutMe. But after i deploy my project i got the error:

404
File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL as well as any file permissions.
For root URLs (like http://example.com/) you must provide an index.html file.

[Read the full documentation](https://help.github.com/pages/) for more information about using GitHub Pages.

Can somebody help me?

**Comment by fury-05** [comment, marked as answer]
Here your folder are wrongly place all the file should be in you root folder i.e. directly in your repo. 

Currently its root\/about-me-app\your files

Also its angular app so I assume that yo have taken care of additional steps involved in deployment.

**Reply by Vatafus** [reply]
Thx for the help.

**Comment by smile199705** [comment]
You are the original page display normal, after the new push to build pages display 404 right?

**Reply by Vatafus** [reply]
Its all good now after I changed the root folder.

**Reply by smile199705** [reply]
Every time a new md document is added, from the local push to the use of the master, github pages will automatically build and publish, but after the update is published, the page will appear every time the problem of incomplete display, I don't know how. Then you only need to manually stop it by yourself, and then it can be displayed normally in the reconstruction. Have you ever encountered such a problem? How was it solved?

**Reply by smile199705** [reply]
My problem, I've solved it. Thank you
